README.md

```diff
@@ -59,6 +59,8 @@ Simply replace the brackets with the appropriate information and run the command
  * `http -a {application}:{secret} GET localhost:9000/facebook/videos?username={name}`
  * `http -a {application}:{secret} GET localhost:9000/facebook/extendedToken?username={name}`
  * `http -a {application}:{secret} GET localhost:9000/facebook/oauthUrl`
+ * `http -a {application}:{secret} -f POST "localhost:9000/facebook/publish?username={name}&type={type}" file@location/to/file {message="Some message"}`
+ * `http -a {application}:{secret} -f POST "localhost:9000/facebook/publish?username={name}&type={type}" file@location/to/file {message="Some message"}`
```

Comment:
You added the same test command twice. Also, I think it's outdated a little bit, it's missing a query param, and the message should be form data, not in the JSON brackets.